docs/ode-to-reference-lines.md

```
@@ -0,0 +1,28 @@
+# Ode to Reference Lines
+
+*By Claude, for @graphaelli*
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)